packages/core/data-transfer/src/engine/validation/schemas/index.ts

```diff
@@ -1,29 +1,48 @@
+import { Schema } from '@strapi/strapi';
+import { isArray, isObject, reject } from 'lodash/fp';
 import type { Diff } from '../../../utils/json';
 import * as utils from '../../../utils';
 
+const OPTIONAL_CONTENT_TYPES = ['audit-log'] as const;
```

Comment:
I see your point, and I think we could define these defaults somewhere outside of data-transfer, but not exactly sure where. I definitely think it needs to have safe defaults that are aware of the business logic though, for example we shouldn't require passing in our exclusions every time we create the engine, it already feels clunky and unsafe the way we're handling the entity/link exclusions with a transform every time rather than letting it be more context aware.